docs/README.md

```diff
@@ -125,37 +123,49 @@ Read the [Streaming Source Ingestion Guide](https://linkedin.github.io/feathr/ho
-![Architecture](images/architecture.png)
+![Architecture Diagram](./images/architecture.png)
 
-## Next Steps
+| Feathr component                | Cloud Integrations                                                          |
+| ------------------------------- | --------------------------------------------------------------------------- |
+| Offline store – Object Store    | Azure Blob Storage, Azure ADLS Gen2, AWS S3                                 |
+| Offline store – SQL             | Azure SQL DB, Azure Synapse Dedicated SQL Pools, Azure SQL in VM, Snowflake |
+| Streaming Source                | Kafka, EventHub                                                             |
+| Online store                    | Azure Cache for Redis                                                       |
+| Feature Registry and Governance | Azure Purview                                                               |
+| Compute Engine                  | Azure Synapse Spark Pools, Databricks                                       |
+| Machine Learning Platform       | Azure Machine Learning, Jupyter Notebook, Databricks Notebook               |
+| File Format                     | Parquet, ORC, Avro, JSON, Delta Lake                                        |
+| Credentials                     | Azure Key Vault                                                             |
 
-### Quickstart
+## 🚀 Roadmap
 
-- [Quickstart for Azure Synapse](quickstart_synapse.md)
+For a complete roadmap with estimated dates, please [visit this page](https://github.com/linkedin/feathr/milestones?direction=asc&sort=title&state=open).
 
-### Concepts
 
-- [Feature Definition](concepts/feature-definition.md)
-- [Feature Generation](concepts/feature-generation.md)
-- [Feature Join](concepts/feature-join.md)
-- [Point-in-time Correctness](concepts/point-in-time-join.md)
+- [x] Support streaming
+- [x] Support common data sources
+- [ ] Support online transformation
+- [ ] Support feature versioning
+- [ ] Support feature monitoring
+- [ ] Support feature store UI, including Lineage and Search functionalities
+- [ ] Support feature data deletion and retention
 
-### How-to-guides
+## 👨‍👨‍👦‍👦 Community Guidelines
 
-- [Azure Deployment](how-to-guides/azure-deployment.md)
-- [Local Feature Testing](how-to-guides/local-feature-testing.md)
-- [Feature Definition Troubleshooting Guide](how-to-guides/troubleshoot-feature-definition.md)
-- [Feathr Expression Language](how-to-guides/expression-language.md)
-- [Feathr Job Configuration](how-to-guides/feathr-job-configuration.md)
+Build for the community and build by the community. Check out [Community Guidelines](CONTRIBUTING.md).
 
-## API Documentation
+## 📢 Slack Channel
 
-- [Python API Documentation](https://feathr.readthedocs.io/en/latest/)
+Join our [Slack channel](https://feathrai.slack.com) for questions and discussions (or click the [invitation link](https://join.slack.com/t/feathrai/shared_invite/zt-1bgiu8eup-yOAKsOOIVGBVjT8B~XMu~A)).
```

Comment:
Might consider change this slack to aka link otherwise a PR is required everytime this link expires